tests/conftest.py

```diff
@@ -42,3 +48,31 @@ def disable_rsa_checks(backend):
     backend._rsa_skip_check_key = True
     yield
     backend._rsa_skip_check_key = False
+
+
+def enable_fips():
```

Comment:
Is there no way to enable this external to the process?

Reply:
This can be done via config but to do that we would need to modify our initialization to read the config by default (it currently does not). That would possibly change behavior for users who don't use our wheels.